### ISSUE TITLE
[Feature request] Add shorthand support for scroll-padding and scroll-margin in enforces-shorthand rule

### ISSUE THREAD

**Issue by Mini-ghost** [francoismassart/eslint-plugin-tailwindcss#432]
**Is your feature request related to a problem? Please describe.**
The `enforces-shorthand` rule doesn’t detect shorthand opportunities for `scroll-padding` and `scroll-margin`.
For example, `scroll-pl-3 scroll-pr-3` could be `scroll-px-3`, but no warning is shown.

**Describe the solution you'd like**
Add shorthand support for `scroll-padding` and `scroll-margin` utilities, just like `p-*` and `m-*`.
For example:

- `scroll-pl-* scroll-pr-*` → `scroll-px-*`
- `scroll-mt-* scroll-mb-*` → `scroll-my-*`
- support negative values like `-scroll-ml-2 -scroll-mr-2` → `-scroll-mx-2`.

I'm happy to contribute a PR to implement and test it.

**Comment by francoismassart** [comment]
@Mini-ghost 
Thank you for reporting the issue.

Bad news, I'll not work anymore on v3

Good news, the bug is fixed on v4, I added your test case

<img width="341" height="115" alt="Image" src="https://github.com/user-attachments/assets/95af19b8-87d6-46c3-bf63-38ac9184625a" />

V4 is almost ready and you can already try it out via:
```
npm i eslint-plugin-tailwindcss@4.0.0-alpha.5
```
